examples/keras_recipes/creating_tfrecords.py

```diff
@@ -0,0 +1,193 @@
+"""
+Title: Creating TFRecords
+Author: [Dimitre Oliveira](https://www.linkedin.com/in/dimitre-oliveira-7a1a0113a/)
+Date created: 2021/02/22
+Last modified: 2021/02/22
+Description: Converting data to TFRecords files.
+"""
+
+"""
+## Reference
+
+- [TFRecord and tf.train.Example](https://www.tensorflow.org/tutorials/load_data/tfrecord)
+
+## Introduction
+
+The TFRecord format is a simple format for storing a sequence of binary records.
+
+Converting your data into TFRecords has many advantages, especially if later, you load it
+using the [tf.data](https://www.tensorflow.org/guide/data) API. The data will take less
+space from disk, it can also be read and processed faster.
+
+Here we will learn how to convert data of different types (image, text, and numeric) into
+TFRecords.
+
+## Dependencies
+"""
+
+import os
+import json
+import pprint
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+"""
+## Downloading the COCO2017 dataset
+
+We will be using the [COCO2017](https://cocodataset.org/) dataset because it has many
+different types of features and will serve as a good example of how to encode different
```

Comment:
We should enumerate the features and their properties

Reply:
Added information about the dataset and its features.